### PR TITLE
L-SMASH 2.9.1

### PR DESCRIPTION
**Pull request by henry0312**

### CODE REVIEW

File: Library/Formula/l-smash.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class LSmash < Formula
   desc "Tool for working with MP4 files"
   homepage "https://l-smash.github.io/l-smash/"
-  url "https://github.com/l-smash/l-smash.git", :tag => "v1.13.2", :shallow => false
+  url "https://github.com/l-smash/l-smash.git", :tag => "v2.9.1", :shallow => false
```
Comment (DomT4) [inline Library/Formula/l-smash.rb]: You'll need to add a `revision` (The upstream commit the tag is based off) here as well now.

```ruby
url "https://github.com/l-smash/l-smash.git", :shallow => false,
     :tag => "v2.9.1",
     :revision => "xyz"
```

Comment (henry0312) [inline Library/Formula/l-smash.rb]: OK :smile:

**Comment by henry0312** [comment]
updated.

**Comment by DomT4** [comment]
Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5fab01529f8bc3ba2e4bac6251f1f5b6156f9c0d. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**